packages/create-vite/template-react-ts/package.json

```
@@ -23,7 +23,7 @@
     "eslint-plugin-react-refresh": "^0.4.7",
     "globals": "^15.6.0",
     "typescript": "^5.5.3",
-    "typescript-eslint": "^7.18.0",
+    "typescript-eslint": "8.0.0",
```

```suggestion
    "typescript-eslint": "^8.0.0",
```

Should this be pinned?

---

Ah, I'd missed that !